addons/l10n_cl/views/report_invoice.xml

```diff
@@ -220,6 +219,57 @@
             <attribute name="t-call">l10n_cl.tax_totals_widget</attribute>
         </t>
 
+        <xpath expr="//div[hasclass('invoice_main')]" position="inside">
+            <t t-if="show_info_box">
+                <div class="row g-0" style="float: right;">
+                    <div class="col-12">
+                        <div class="d-inline-block ms-auto" style="width: 440px;">
```

Comment:
Style should be avoided at all cost for different reason: 
- Difficult to maintain, using class we can actually modify the class for everything in the code to have consistency, here you cannot
- Readability: here you have just one styling so ok but you can have a lot and it will become a nightmare
- Priority: let's say we have a min height in one of the class, the styling will take priority on everything, which you could lead to problematic behavior

So always use bootstrap class instead :smile: